automation/greetings.yml

```diff
@@ -9,5 +9,5 @@ jobs:
     - uses: actions/first-interaction@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        issue-message: 'Message that will be displayed on users'' first issue'
-        pr-message: 'Message that will be displayed on users'' first pr'
+        issue-message: 'Message that will be displayed on users'
+        pr-message: 'Message that will be displayed on users'
```

Comment:
```suggestion
        issue-message: 'Message that will be displayed on users first issue'
        pr-message: 'Message that will be displayed on users first pull request'
```

Reply:
Hi @andymckay - Indeed, we don't want the sentence to trail off. However, now it's gramatically incorrect. How about:

issue-message: 'Message that will be displayed on the first issue for that user'
pr-message: 'Message that will be displayed on the first pr for that user'

Reply:
Sure that works for me 👍🏾